setup.py

```diff
@@ -0,0 +1,14 @@
+"""Setup file to run this network_simulator package.
```

Comment:
aside: not sure whether we need a setup.py for this project, but it's not a bad idea.